package.json

```diff
@@ -20,11 +20,13 @@
     "storybook:build": "storybook build -c storybook -o storybook/public --docs",
     "test:storybook": "STORIES=ignoreBrandMoment test-storybook --config-dir storybook",
     "jest:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --no-cache",
-    "lint": "yarn lint:ts --max-warnings=0 && yarn lint:format && yarn lint:styles",
-    "lint:ts": "yarn eslint . -c .eslintrc.js --ext .ts,.tsx",
+    "lint": "yarn lint:ts && yarn lint:md && yarn lint:format && yarn lint:styles",
+    "eslint-config": "yarn eslint -c .eslintrc.js --max-warnings=0",
+    "lint:ts": "yarn eslint-config '**/*.{ts,tsx}'",
+    "lint:md": "yarn eslint-config '**/*.{md,mdx}'",
```

Comment:
Couldn't use `--ext .md,.mdx` because the code snippets within are apparently embedded within (`*.{md,mdx}/**`), so if I applied the `ext` flag, the linting is missed.

Reply:
Alt I could merge them into the same command.